src/applications/simple-forms/shared/components/ConfirmationPageView.v2.jsx

```diff
@@ -61,30 +76,6 @@ export const ConfirmationPageView = props => {
     [alertRef],
   );
 
-  const form = useSelector(state => state.form);
```

Comment:
goal: keep this file as minimal as possible and import things from (eventually forms-system)